core/domain/fs_domain.py

```diff
@@ -293,7 +347,7 @@ def isfile(self, filepath):
         self._check_filepath(filepath)
         return self._impl.isfile(filepath)
 
-    def open(self, filepath):
+    def open(self, filepath: str) -> FileStream | Any:
```

Comment:
Why `Any`?

Reply:
because it is returning any instead of returning filestream

Reply:
When and why?

Reply:
![image](https://user-images.githubusercontent.com/37413698/152553791-f77c3a94-62b2-4cae-80e9-2948ac16dedb.png)
This is the error that I am getting if I remove |Any

Reply:
Or use `FileStream | None`